.github/workflows/pytest.yml

```diff
@@ -0,0 +1,79 @@
+name: pytest
+
+on: [pull_request, push]
+
+jobs:
+  test:
+    env:
+      # we need to turn "3.6" into 36 to build the conda requirement filename,
+      # but there's no good way to do that via string manipulation.  So set up
+      # a map that will be parsed as JSON later:
+      VERSIONMAP: '{"3.6": "36", "3.7": "37", "3.8": "38", "3.9": "39"}'
+
+    strategy:
+      fail-fast: false  # don't cancel other matrix jobs when one fails
+      matrix:
+        # use macos-10.15 instead of macos-latest for py3.6 support
+        os: [ubuntu-latest, macos-10.15, windows-latest]
+        python-version: [3.6, 3.7, 3.8, 3.9]
+        environment-type: [conda, bare]
+        suffix: ['']  # placeholder as an alternative to "-min"
+        include:
+          - os: ubuntu-latest
+            python-version: 3.6
+            environment-type: conda
+            suffix: -min
+        exclude:
+          - os: macos-10.15
+            environment-type: conda
+          - os: windows-latest
+            environment-type: bare
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
There's now a v3. Probably no meaningful change but I think we might as well use the newest as we set this up.